pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -155,54 +155,61 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 
 	// Reuse LimitRanger logic to apply limit/req defaults from the project. Ignore validation
 	// errors, assume that LimitRanger will run after this plugin to validate.
-	glog.V(5).Infof("%s: initial pod limits are: %#v", api.PluginName, pod.Spec.Containers[0].Resources)
+	glog.V(5).Infof("%s: initial pod limits are: %#v", api.PluginName, pod.Spec)
```

Comment:
Not that I'm aware of, since this can be across multiple containers, init containers, requests, and limits (and potentially annotations).  Might be nice in the future.